flink-libraries/flink-table/src/test/scala/org/apache/flink/api/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -1101,6 +1101,45 @@ class ScalarFunctionsTest extends ExpressionTestBase {
       "true")
   }
 
+  @Test
+  def testInExpressions(): Unit = {
+    testTableApi(
```

Comment:
@twalthr Unfortunately, add support for POJO and tuples as a standard types of data in table API is too complicated task and out of scope this task. Adding of new composite types to API more wide than add IN operator, it will impact all process of validation and execution of SQL statements. It would be better create epic for introduce new types.
I have researched it and I think I would be better commit IN operator functionality without support tuples and POJOs. Otherwise, It would become infinite task.